### PR TITLE
Add npm install to release steps

### LINKED ISSUE #1517: HeroBanner: changes when releasing another component
Often if i release, i got these changes in heroBanner. Dont know why.

image

### PR DESCRIPTION
**Pull request by raphaellueckl**
resolves #1517

Oftentimes we have package-lock files when we run `npm install`. Those are not properly updated during the release and merging of the release-branch. 

This PR makes sure that those files are also generated (a release without making sure that npm install already happened shouldn't be possible anyways).

### CODE REVIEW

**Comment by danyball** [comment]
i do not use the release script. i am using the introduction in contribution.md ?!

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -118,19 +118,20 @@ storiesOf('Molecules/Top content bar', module)
 
 ## How-to release a component
 
+**Do not use the release scripts: ~~`npm run release`~~ or ~~`npm run prerelease`~~**
```
Comment (danyball) [inline CONTRIBUTION.md:121]: So, lets delete these scripts....!

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -118,19 +118,20 @@ storiesOf('Molecules/Top content bar', module)
...
+
 - Ensure you have an npm account under the `@axa-ch` organisation
 - Create a new _release branch_ that follows this pattern: `release/<component-name>`
 - Clean the main and components dependencies:
   - `npx lerna clean`
   - `rm -rf node_modules`
 - Run `npm install`
```
Comment (danyball) [inline CONTRIBUTION.md]: I have package-lock changes at this point

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -118,19 +118,20 @@ storiesOf('Molecules/Top content bar', module)
...
+- Manually add your component to `lerna.json` and remove all not-to-be-released components from the same file
+- Commit and push your lerna.json changes to the release branch
 - Ensure you are logged in to npm (`npm login`)
 - Ensure you have the necessary credentials to push to the Githup repo via commandline commands (the following lerna commands do this automatically)
 - Run `npm run test && npx lerna version && npx lerna publish from-package`
 - Lerna is going to ask you how to update the changed packages. Follow the instructions and keep in mind the semver rules: https://semver.org/
 - Lerna will update the component `package.json` with new version numbers and auto-commit/push those
-- Manually undo the removal of all not-to-be-released components in lerna.json, commit and push again to the release branch
-- merge the release branch to the `develop` branch in order to preserve the new version numbers
+- Manually undo the removal of all not-to-be-released components in lerna.json, run `npm install` and commit and push again to the release branch (including all the package-lock.json files)
```
Comment (danyball) [inline CONTRIBUTION.md]: Why to run npm i again?

**Review by danyball — CHANGES_REQUESTED** [review]

File: CONTRIBUTION.md (hunk 1 above)
Comment (raphaellueckl) [reply CONTRIBUTION.md:121]: Usually I would agree, but AFAIK luca is having a plan here (and already uses them).

File: CONTRIBUTION.md (hunk 2 above)
Comment (raphaellueckl) [reply CONTRIBUTION.md]: Then this is because of someone else that didn't do it correctly.

File: CONTRIBUTION.md (hunk 3 above)
Comment (raphaellueckl) [reply CONTRIBUTION.md]: I added info here.

File: CONTRIBUTION.md [hunk 4]
```diff
@@ -118,19 +118,20 @@ storiesOf('Molecules/Top content bar', module)
...
+- Manually add your component to `lerna.json` and remove all not-to-be-released components from the same file.
+- Commit and push your lerna.json changes to the release branch.
+- Ensure you have the necessary credentials to push to the Githup repo via commandline commands (the following lerna commands do this automatically).
 - Run `npm run test && npx lerna version && npx lerna publish from-package`
 - Lerna is going to ask you how to update the changed packages. Follow the instructions and keep in mind the semver rules: https://semver.org/
-- Lerna will update the component `package.json` with new version numbers and auto-commit/push those
-- Manually undo the removal of all not-to-be-released components in lerna.json, commit and push again to the release branch
-- merge the release branch to the `develop` branch in order to preserve the new version numbers
+- Lerna will update the component `package.json` with new version numbers and auto-commit/push those.
+- Manually undo the removal of all not-to-be-released components in lerna.json, run `npm install` and commit the generated package-lock files to the release branch.
```
Comment (danyball) [inline CONTRIBUTION.md:136]: Why npm install again?

Comment (raphaellueckl) [reply CONTRIBUTION.md:136]: I added info here.

**Review by danyball — APPROVED** [review]

**Pull request merged**